modules/java.nix

```diff
@@ -1,19 +1,26 @@
-{ pkgs, ... }: let 
+{ pkgs, ... }:
+
+let
+  graalvm = pkgs.graalvm17-ce;
+
   graal-compile-command = "${pkgs.graalvm17-ce}/bin/javac -classpath .:target/dependency/* -d . $(find . -type f -name '*.java')";
-in {
+in
+
+{
   name = "Java Tools";
   version = "1.0";
 
-  packages = with pkgs; [
-    maven
+  packages = [
+    graalvm
```

Comment:
not necessary, but it better aligns with other modules where the language package is included for user access via cli